package.json

```diff
@@ -6,6 +6,7 @@
     "uglify-js": "2.4.24"
   },
   "devDependencies": {
-    "jshint": "^2.7.0"
+    "jshint": "^2.7.0",
+    "edx-custom-a11y-rules": "edx/edx-custom-a11y-rules"
```

Comment:
I'm wondering if this needs to be published in some grand npm location? I notice this isn't pulling it in via github, for example.